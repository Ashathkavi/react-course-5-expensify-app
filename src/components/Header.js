

import React from 'react';
import { NavLink, Outlet} from 'react-router-dom';


const Header = (props) => (
    <div>
        <h1>Expensify</h1>
        <nav>
            <NavLink className={({ isActive }) => isActive ? "is-active" : ""} to="/dashboard">Dashborad</NavLink>
            <NavLink className={({ isActive }) => isActive ? "is-active" : ""} to="/create">Create</NavLink>
            <NavLink className={({ isActive }) => isActive ? "is-active" : ""} to="/help">Help</NavLink>
        </nav>
        <Outlet/>
    </div>
)
export default Header