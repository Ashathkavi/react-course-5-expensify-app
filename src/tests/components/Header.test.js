import ReactShallowRenderer from 'react-test-renderer/shallow'
import React from 'react'
import Header from '../../components/Header'


test('shiould render hesder correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header/>)
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})




