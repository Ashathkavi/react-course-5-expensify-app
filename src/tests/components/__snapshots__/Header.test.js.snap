// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shiould render hesder correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <nav>
    <NavLink
      className={[Function]}
      to="/dashboard"
    >
      Dashborad
    </NavLink>
    <NavLink
      className={[Function]}
      to="/create"
    >
      Create
    </NavLink>
    <NavLink
      className={[Function]}
      to="/help"
    >
      Help
    </NavLink>
  </nav>
  <Outlet />
</div>
`;
